src/components/call-to-action-overlay/call-to-action-overlay.tsx

```diff
@@ -57,30 +58,34 @@ const CallToActionOverlay = withText({closeLabel: 'overlay.close'})(
   connect(mapStateToProps)(
     ({title, description, buttons, onClose, onClick, closeLabel, descriptionLines, sizeClass}: CallToActionOverlayProps): any => {
       return (
-        <div data-testid="call-to-action-overlay" className={`${styles.callToActionOverlay} ${sizeClass}`}>
-          <div className={styles.closeButton} data-testid="call-to-action-overlay-close-button">
-            <Button
-              onClick={onClose}
-              type={ButtonType.borderless}
-              size={ButtonSize.medium}
-              tooltip={{label: closeLabel!}}
-              ariaLabel={closeLabel}
-              icon={'close'}
-            />
-          </div>
-          <div className={styles.content}>
-            <div className={styles.title}>
-              <TextWithTooltip text={title || ''} numberOfLines={1} />
+        <div role="alertdialog" data-testid="call-to-action-overlay" className={`${styles.callToActionOverlay} ${sizeClass}`}>
+          <FocusTrap active>
```

Comment:
I've found that playkit-js-ui already has focus-trap utility, could we re-use it in CTA as well?
https://github.com/kaltura/playkit-js-ui/blob/master/src/utils/popup-keyboard-accessibility.tsx#L9

usages
https://github.com/kaltura/playkit-js-info/blob/master/src/components/info/index.tsx#L47
https://github.com/kaltura/playkit-js-moderation/blob/master/src/components/moderation/moderation.tsx#L84

Reply:
interesting, will check it out. thanks.